alicloud/resource_alicloud_vpc.go

```diff
@@ -5,6 +5,7 @@ import (
 	"strings"
 	"time"
 
+	"github.com/denverdino/aliyungo/ecs"
```

Comment:
I don't suggest to use the sdk "github.com/denverdino/aliyungo/ecs" and it will be removed from provider.

Reply:
What do you suggest to use instead of the sdk "github.com/denverdino/aliyungo/ecs"?